### PR TITLE
Feature/controllers

### PR DESCRIPTION
**Pull request by Benawi**
### General requirements
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
### Ruby requirements
- Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).
### Project requirements
-  [x] Make sure that your app handles the URLs used in the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#project-requirements)
- [x] For each URL you should create:
  - A route.
  - An action in the correct controller.
  - A view file.
- [x]  View files should include only placeholders written in plain HTML,
- [x]  Make sure that you added code to handle only the 4 required URLs.
[root](https://github.com/Benawi/Blog-App/assets/21217148/6b10d5a4-35c0-4ae6-a1aa-a545bff7ad21)
[/users/2](https://github.com/Benawi/Blog-App/assets/21217148/64f774da-3fcc-4270-9da6-74fc16fcf5ad)
[/users/2/posts](https://github.com/Benawi/Blog-App/assets/21217148/50965d5d-d85e-4310-b0ad-1ab25e953083)
[/users/2/posts/2](https://github.com/Benawi/Blog-App/assets/21217148/bda5ecbd-243e-46d6-ac04-16c0392f5858)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  root 'users#index'
   # Defines the root path route ("/")
   # root "articles#index"
+  
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create]
+  end
```
Comment (gemmen29) [inline config/routes.rb]: - [ ] Please make sure that your code only handles the 4 required URLs as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.). You added an extra route for post creation which does not exist in the requirement of that project. Kindly remove it.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Benawi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
## Required Changes ♻️
- [x] Routes updated as per requested

File: config/routes.rb (hunk 1 above)
Comment (Abenezer-Tilahun) [reply config/routes.rb]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Benawi 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**